pyproject.toml

```diff
@@ -42,22 +33,126 @@ classifiers = [
     "Topic :: System :: Distributed Computing",
 ]
 
-[project.urls]
-Homepage = "https://github.com/locustio/locust"
-Documentation = "https://docs.locust.io/"
-Code = "https://github.com/locustio/locust"
-"Help/Questions" = "https://stackoverflow.com/questions/tagged/locust"
-"Issue tracker" = "https://github.com/locustio/locust/issues"
-
-
-[project.readme]
-file = "README.md"
-content-type = "text/markdown"
+packages = [{ include = "locust" }]
+include = [
+    "locust/webui/dist/**/*",
+    "locust/_version.py",
+    { path = "poetry.lock", format = [
+        "sdist",
+        "wheel",
+    ] },
+]
+exclude = [
+    "locust/webui/public",
+    "locust/webui/src",
+    "locust/webui/*.*",
+    "locust/test",
+]
 
-[tool.setuptools]
-zip-safe = false
-license-files = ["LICENSE"]
-include-package-data = false
+[tool.poetry.build]
+script = "build.py"
+generate-setup-file = false
+
+[tool.poetry.urls]
+"Help/Questions" = "https://github.com/orgs/locustio/discussions/"
```

Comment:
Can you make the first link to the docs?

Reply:
The repo/docs URLs tend to be first-class citizens in the file, and are listed under the package itself (ln16)

I think these are mostly "other unspecified URLs" - would you want one of these to be docs also?

(sorry for RTD spam builds, think we're good now)

Reply:
Ah. No that should be fine then.